ui/src/admin/constants/tableSizing.js

```diff
@@ -0,0 +1,27 @@
+export const USERS_TABLE = {
```

Comment:
I suppose I could achieve the same effect using a series of styles. In retrospect I started by setting widths inline in all the tables, then remembered "no magic numbers" so I made them into consts, then noticed I had a lot of them so I put them all in one file. Felt organized and nice having controls for that all in one place. Now that I look at it there is no distinct advantage to doing this in JS instead of CSS